ale_linters/python/ruff.vim

```diff
@@ -48,7 +48,7 @@ function! ale_linters#python#ruff#GetCommand(buffer, version) abort
     " NOTE: ruff version `0.0.69` supports liniting input from stdin
     return ale#Escape(l:executable) . l:exec_args
     \   . ale#Pad(ale#Var(a:buffer, 'python_ruff_options'))
-    \   . ' --format text'
+    \   . ' --output-format text'
```

Comment:
can we add a configuration flag to choose which flag to use? that would support 0.0.x and 0.1.0 users.

Reply:
or update documentation somewhere along with this to detail that >=0.1.0 is required. Which feels a bit drastic to force all users of ALE to upgrade all code bases to a version of ruff.

Reply:
If we choose to support earlier versions of ruff, I'd rather see an if-statement on the version here, so the user will not have to bother digging into our docs, find this configuration variable and set it correctly, just to have ruff work.

Reply:
I'll implement the conditional. If the installed Ruff version is lower than `0.1.0` ALE with use the deprecated `--format` flag. Otherwise it will use the new `--output-format` flag